assets/js/hooks/Mapper/components/mapInterface/widgets/RoutesWidget/RoutesWidget.module.scss

```
@@ -6,10 +6,10 @@
   column-gap: 3px;
   row-gap: 2px;
 
-  font-size: 12px;
+  font-size: var(--rf-routes-widget-font-size, 12px);
```

Stylelint (17.14.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)